R/dt_summary.R

```diff
@@ -191,14 +191,20 @@ dt_summary_build <- function(data,
     agg_funs <- fns %>% lapply(rlang::as_closure)
 
     summary_dfs_data <-
+
+      seq_along(agg_funs) %>%
       lapply(
-        seq(agg_funs), function(j) {
+        function(j) {
+
+          label <- labels[j]
+          agg_fun <- agg_funs[[j]]
+
           select_data_tbl %>%
             dplyr::filter(groupname %in% groups) %>%
             dplyr::group_by(groupname) %>%
-            dplyr::summarize_all(.funs = agg_funs[[j]]) %>%
+            dplyr::summarize_all(.funs = agg_fun) %>%
             dplyr::ungroup() %>%
-            dplyr::mutate(rowname = labels[j]) %>%
+            dplyr::mutate(rowname = label) %>%
```

Comment:
This is still not totally safe; if `label` is the name of a column in the user data you could have a problem. Investigate using `!!label` to force evaluation without table overscope.